### PR TITLE
Color offsets update (HaxeFlixel/flixel#1705)

### PR DESCRIPTION
**Pull request by larsiusprime**

### CODE REVIEW

File: flixel/addons/display/FlxSkewedSprite.hx [hunk 1]
```diff
@@ -83,10 +83,10 @@ class FlxSkewedSprite extends FlxSprite
 			
 			getScreenPosition(_point, camera).subtractPoint(offset);
 			
-			var cr:Float = colorTransform.redMultiplier;
-			var cg:Float = colorTransform.greenMultiplier;
-			var cb:Float = colorTransform.blueMultiplier;
-			
+			var croff:Float = colorTransform.redOffset;
```
Comment (Gama11) [inline flixel/addons/display/FlxSkewedSprite.hx]: These local variables seem to be unused?

File: flixel/addons/display/FlxSpriteOrigin.hx [hunk 2]
```diff
@@ -55,10 +55,10 @@ class FlxSpriteOrigin extends FlxSprite
 			
 			getScreenPosition(_point, camera).subtractPoint(offset);
 			
-			var cr:Float = colorTransform.redMultiplier;
-			var cg:Float = colorTransform.greenMultiplier;
-			var cb:Float = colorTransform.blueMultiplier;
-			
+			var croff:Float = colorTransform.redOffset;
```
Comment (Gama11) [inline flixel/addons/display/FlxSpriteOrigin.hx]: Same here.

File: flixel/addons/display/shapes/FlxShapeArrow.hx [hunk 3]
```diff
@@ -111,6 +111,9 @@ class FlxShapeArrow extends FlxShape
 		var canvasWidth:Int = Std.int(Math.max(shapeWidth, arrowSize*2) + strokeBuffer);
 		var canvasHeight:Int = Std.int(Math.max(shapeHeight, arrowSize*2) + strokeBuffer);
 		
+		if (canvasWidth % 2 == 1) { canvasWidth += 1; }
```
Comment (Gama11) [inline flixel/addons/display/shapes/FlxShapeArrow.hx]: Unrelated change?

**Comment by larsiusprime** [comment]
Fixed
